crates/bevy_render/src/settings.rs

```diff
@@ -3,6 +3,7 @@ use crate::renderer::{
 };
 use std::borrow::Cow;
 
+use wgpu::MemoryHints;
```

Comment:
This needs to go into the pub use right below it so it's re-exported, otherwise iirc you won't be able to set it in a 3rd party bevy app.